### DISCUSSION TITLE
[濯水]是我不懂现代艺术?

### DISCUSSION THREAD

**Discussion by Iincan** [share121/inter-knot#1146] [其他]
![13526b8b99bb30269f0dc4ba79aa7781](https://github.com/user-attachments/assets/9b8157c3-b601-46e2-96e1-50e63ddfac7f)
最近去看展。旧都陷落时，芭莱大厦被空洞侵蚀的程度还没有现在这么深。空洞调查员紧急抢救出一批展品。
看完只想说，调查员冒着生命危险抢救这种东西，真的有必要吗?
《抱邦布的男人(们)》《以骸背上的芭菜双子》是我最不喜欢的作品，感觉只是戏仿名作，顺便拍下芭莱老板的马屁。作者是金斯顿，伯德。据说这个人没有经过学院派的训练，只是一个临摹名画的画工，这都可以吗?

**Comment by DZY-QAQ** [comment]
很好的文章，使芭莱老板的大厦旋转

**Comment by zhangyj0523** [comment]
遇到十二分街道本地人了

**Comment by Gmadelyn7** [comment]
非也，这些美妙的画作，无疑是艺术领域中一种充满创意与原创性的探索。艺术家们以其精湛的技艺，巧妙地将个人风格与经典作品融合，不仅向经典致敬，也在与历史对话，赋予作品跨越时代的文化意义。这些作品在视觉冲击力上给人以强烈的震撼，通过对比和反差，引导观众从新的角度审视经典，感受艺术的无限可能。同时，戏仿作品中往往蕴含的幽默与讽刺元素，为艺术创作增添了更多的趣味性和思考空间，使得观众在欣赏的同时，也能体会到艺术家对当代社会的独特见解。此外，这些作品还常常蕴含着深层的哲学思考，艺术与哲学的结合，引发观众对艺术、生活乃至存在的深层次反思。总而言之，戏仿名作的画作是艺术家智慧和情感的结晶，它们不仅是对经典艺术形式的一种致敬，也是对艺术创新精神的一种展现。

**Comment by 112233-oss** [comment]
豪看\(^o^)/~
